### PR TITLE
Update de.json

### PR DESCRIPTION
**Pull request by cebe**
Not sure of these are 100% correct, could need some more context or description of where they appear and what they do if you check or click.

### CODE REVIEW

File: languages/de.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "About <AppName>" : "Über <AppName>",
-  "Bring <AppName> Window to Front" : "<AppName> nach vorne bringen",
-  "Bring All to Front" : "Alle nach vorne bringen",
+  "Bring <AppName> Window to Front" : "<AppName> in den Vordergrund bringen",
+  "Bring All to Front" : "Alle in den Vordergrund bringen",
```
Comment (cebe) [inline languages/de.json]: `in den Vordergrund` is more precize than `nach vorne`.

File: languages/de.json [hunk 2]
```diff
@@ -50,8 +50,8 @@
   "Text Replacement" : "Text ersetzen",
   "Transformations" : "Transformationen",
   "Undo" : "Widerrufen",
-  "Use Selection for Find" : "Auswahl suchen",
-  "Use Selection for Replace" : "Auswahl für Ersetzen übernehmen",
+  "Use Selection for Find" : "In Auswahl suchen",
+  "Use Selection for Replace" : "In Auswahl ersetzen",
```
Comment (cebe) [inline languages/de.json]: I am missing context on this one but it sounds like when I click this, I will do a find or search not in whole document but in the selection, right?

File: languages/de.json (hunk 1 above)
Comment (maremmle) [inline languages/de.json]: The translation should match Apple's. In Pages it says ``nach vorne``.

File: languages/de.json (hunk 2 above)
Comment (maremmle) [inline languages/de.json]: Again, according to Apple's Pages ``"Auswahl für Ersetzen übernehmen"`` should be right.

**Pull request closed without merging**